src/cpp/core/include/core/r_util/RTokenizer.hpp

```diff
@@ -511,6 +511,28 @@ inline bool canContinueStatement(const RToken& rToken)
            canOpenArgumentList(rToken);
 }
 
+inline bool isSymbolNamed(const RToken& rToken,
+                          const std::wstring& name)
+{
+   // For strings, check if the content within the quotes
+   // is equal to the name provided. TODO: handle escaped
```

Comment:
Okay, fine re: not handling the quotes inside a quoted function.

On Wed, Apr 8, 2015 at 2:12 PM, Kevin Ushey <notifications@github.com>
wrote:

> In src/cpp/core/include/core/r_util/RTokenizer.hpp
> <https://github.com/rstudio/rstudio/pull/380#discussion_r27996977>:
>
> > @@ -511,6 +511,28 @@ inline bool canContinueStatement(const RToken& rToken)
> >             canOpenArgumentList(rToken);
> >  }
> >
> > +inline bool isSymbolNamed(const RToken& rToken,
> > +                          const std::wstring& name)
> > +{
> > +   // For strings, check if the content within the quotes
> > +   // is equal to the name provided. TODO: handle escaped
>
> That note implies that we don't handle someone naming a symbol something
> like "a\"function" <- function(x) { ... }, which not a big deal IMO.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rstudio/pull/380/files#r27996977>.
>